Tasks/RunJMeterLoadTest/task.json

```diff
@@ -0,0 +1,87 @@
+{
+  "id": "F20661EB-E0F7-4AFD-9A86-9FE9D1A93382",
+  "name": "ApacheJMeterLoadTest",
+  "friendlyName": "Cloud-based Apache JMeter Load Test",
+  "description": "Runs the Apache JMeter load test in cloud",
+  "helpMarkDown": "This task can be used to trigger an Apache JMeter load test in cloud using Visual Studio Team Services. [Learn more](http://go.microsoft.com/fwlink/?LinkId=784929).",
+  "category": "Test",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 1,
+    "Minor": 0,
+    "Patch": 0
+  },
+  "demands": [
+    "msbuild",
```

Comment:
I don't completely understand JMeter. But is the demands right, does it need msbuild?